processes/consumer/kafka.go

```diff
@@ -124,6 +124,11 @@ func StartConsumer(ctx context.Context) {
 					continue
 				}
 
+				if len(msg.Value()) == 0 {
+					slog.With(logFields...).Info("found a tombstone message, skipping...")
```

Comment:
I think you can do `slog.Info("found a tombstone message, skipping...", logFields...))`, the other ones use `With` because we're sending along an error too.

Reply:
cool